tests/test_routes.py

```diff
@@ -630,7 +630,7 @@ def test_update_items_400(self):
 
         # update item with missing name
         no_name_data = data.copy()
-        no_name_data.pop("name")
+        no_name_data["name"] = None
```

Comment:
I think we should remain testing with `no_name_data.pop("name")` because it is possible that API caller does not include "name" in the request body, and therefore we need this test case.